tools/ci/download_bin.sh

```diff
@@ -54,7 +57,12 @@ download_artifact()
     if [ ! -z "${download_gm}" -a -z "${is_gm}" ] || [ -z "${download_gm}" -a ! -z "${is_gm}" ]
```

Comment:
What's the function of `is_gm`, can it be deleted?

Reply:
no, it's a variable.